spec/factory_bot/evaluator_class_definer_spec.rb

```diff
@@ -13,7 +13,7 @@
   end
 
   it "evaluates the block in the context of the evaluator" do
-    dependency_attribute = double(
+    double(
```

Comment:
I we aren't ever using the variable, I'm guessing we can just get rid of this double entirely

Reply:
👍